scripts/combine/setupCombine.py

```diff
@@ -435,7 +433,6 @@ def setup(args, inputFile, fitvar, xnorm=False):
                     mirrorDownVarEqualToNomi=False
                     groupName = "muon_eff_syst"
                     splitGroupDict = {f"{groupName}_{x}" : f".*effSyst.*{x}" for x in list(effTypesNoIso + ["iso"])}
-                    splitGroupDict[groupName] = ".*effSyst.*" # add also the group with everything
```

Comment:
This line is needed to have also the global group for eff_syst. It must be kept. Same below for eff_stat, otherwise you only get the single pieces.